connexion/utils.py

```diff
@@ -205,3 +206,40 @@ def iscorofunc(func):
     else:  # pragma: 3 no cover
         # there's no asyncio in python 2
         return False
+
+
+def yamldumper(openapi):
+    """
+    Returns a nicely-formatted yaml spec.
+    :param openapi: a spec dictionary.
+    :return: a nicely-formatted, serialized yaml spec.
+    """
+    def should_use_block(value):
+        for c in u"\u000a\u000d\u001c\u001d\u001e\u0085\u2028\u2029":
```

Comment:
oops, I meant on this line..

Reply:
Taken from here. https://fossies.org/linux/ansible/lib/ansible/plugins/callback/yaml.py will add comment asap.